### DISCUSSION TITLE
room 네이밍

### DISCUSSION THREAD

**Discussion by dev210202** [OPGG-HACKTHON/mobile-b-android#5] [General]
Room 라이브러리와 혼동될 수 있을 것 같아서 방에 관련된 네이밍을 room이 아닌 다른것으로 바꿔야 하지않을까 싶습니다

**Comment by jisungbin** [comment]
근데 이 프로젝트에서 room 라이브러리가 필요한가요? 데이터베이스에 저장할게 없지 않나요..?

**Reply by dev210202** [reply]
그렇네요 로그인이나 저장할것이 없어서 당장은 필요없을것 같네요 그냥 room으로 해도 될것같아요
